tools/CustomMSBuild/ODataRuleSets.ruleset

```diff
@@ -115,22 +115,22 @@
     <Rule Id="CA1709" Action="Error" />
     <Rule Id="CA1710" Action="Error" />
     <Rule Id="CA1711" Action="Error" />
-    <Rule Id="CA1712" Action="Error" />
+    <Rule Id="CA1712" Action="Warning" />
     <Rule Id="CA1713" Action="Error" />
     <Rule Id="CA1714" Action="Error" />
     <Rule Id="CA1715" Action="Error" />
     <Rule Id="CA1716" Action="Error" />
     <Rule Id="CA1717" Action="Error" />
     <Rule Id="CA1719" Action="Error" />
-    <Rule Id="CA1720" Action="Error" />
+    <Rule Id="CA1720" Action="Info" />

```

Comment:
Why not warning?

Reply:
Some of these rules were changed to `Info` when we upgraded to FxCop analyzers. Let me manually adjust them to `Warnings`

Reply:
For CA1720: Identifiers should not contain type names.
However, in EdmLib, we have Enum properties with type name in `EdmPrimitiveTypeKind`